stdlib/public/Platform/TiocConstants.swift

```diff
@@ -0,0 +1,131 @@
+/// set pgrp of tty
+public let TIOCSPGRP:UInt = 0x80047476
+/// output queue size
+public let TIOCOUTQ:UInt = 0x40047473
+/// simulate terminal input
+public let TIOCSTI:UInt = 0x80017472
+/// void tty association
+public let TIOCNOTTY:UInt = 0x20007471
+/// stop output, like ^S
+public let TIOCSTOP:UInt = 0x2000746f
+/// start output, like ^Q
+public let TIOCSTART:UInt = 0x2000746e
+/// set all modem bits
+public let TIOCMSET:UInt = 0x8004746d
+/// bis modem bits
+public let TIOCMBIS:UInt = 0x8004746c
+/// bic modem bits
+public let TIOCMBIC:UInt = 0x8004746b
+/// get all modem bits
+public let TIOCMGET:UInt = 0x4004746a
+/// remote input editing
+public let TIOCREMOTE:UInt = 0x80047469
+/// get window size
+public let TIOCGWINSZ:UInt = 0x40087468
+/// set window size
+public let TIOCSWINSZ:UInt = 0x80087467
+/// pty: set/clr usr cntl mode
+public let TIOCUCNTL:UInt = 0x80047466
+/// simulate ^T status message
+public let TIOCSTAT:UInt = 0x20007465
+/// 4.2 compatibility
+public let TIOCSCONS:UInt = 0x20007463
+/// become virtual console
+public let TIOCCONS:UInt = 0x80047462
+/// become controlling tty
+public let TIOCSCTTY:UInt = 0x20007461
+/// pty: external processing
+public let TIOCEXT:UInt = 0x80047460
+/// pty: generate signal
+public let TIOCSIG:UInt = 0x2000745f
+/// wait till output drained
+public let TIOCDRAIN:UInt = 0x2000745e
+/// modem: set wait on close
+public let TIOCMSDTRWAIT:UInt = 0x8004745b
+/// modem: get wait on close
+public let TIOCMGDTRWAIT:UInt = 0x4004745a
+/// enable/get timestamp of last input event
+public let TIOCTIMESTAMP:UInt = 0x40107459
+/// enable/get timestamp of last DCd rise
+public let TIOCDCDTIMESTAMP:UInt = 0x40107458
+/// set ttywait timeout
+public let TIOCSDRAINWAIT:UInt = 0x80047457
+/// get ttywait timeout
+public let TIOCGDRAINWAIT:UInt = 0x40047456
+/// download microcode to DSI Softmodem
+public let TIOCDSIMICROCODE:UInt = 0x20007455
+/// grantpt(3)
+public let TIOCPTYGRANT:UInt = 0x20007454
+/// ptsname(3)
+public let TIOCPTYGNAME:UInt = 0x40807453
+/// unlockpt(3)
+public let TIOCPTYUNLK:UInt = 0x20007452
+
+// From ioctl_compact.h
+
+/// hang up on last close
+public let TIOCHPCL:UInt = 0x20007402
+/// get parameters -- gtty
+public let TIOCGETP:UInt = 0x40067408
+/// set parameters -- stty
+public let TIOCSETP:UInt = 0x80067409
+/// as above, but no flushtty
+public let TIOCSETN:UInt = 0x8006740a
+/// set special characters
+public let TIOCSETC:UInt = 0x80067411
+/// get special characters
+public let TIOCGETC:UInt = 0x40067412
+/// bis local mode bits
+public let TIOCLBIS:UInt = 0x8004747f
+/// bic local mode bits
+public let TIOCLBIC:UInt = 0x8004747e
+/// set entire local mode word
+public let TIOCLSET:UInt = 0x8004747d
+/// get local modes
+public let TIOCLGET:UInt = 0x4004747c
+/// set local special chars
+public let TIOCSLTC:UInt = 0x80067475
+/// get local special chars
+public let TIOCGLTC:UInt = 0x40067474
+
+#endif // os(OSX) || os(iOS) || os(tvOS) || os(watchOS) || os(FreeBSD
```

Comment:
Please remove the trailing comment on `#endif`, we don't use them in Swift code.